src/org/labkey/test/pages/ReactAssayTypeSelectPage.java

```diff
@@ -0,0 +1,86 @@
+package org.labkey.test.pages;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ReactAssayTypeSelectPage<EC extends ReactAssayTypeSelectPage.ElementCache> extends WebDriverComponent<EC>
+{
+    private final WebElement el;
+    private final WebDriver driver;
+
+    public ReactAssayTypeSelectPage(WebDriver driver)
+    {
+        this.driver = driver;
+        el = Locator.id("app").findElement(this.driver); // Full page component
```

Comment:
It's preferable to use the finder pattern in the component template- that gives calling code the option to use waitFor to synchronize until the component is present

Reply:
Updating to extend from LabKeyPage made this not necessary.  Removed